### PR TITLE
71 - Add logging for Google requests

### PR DESCRIPTION
**Pull request by abubakarsohail**
This pull request adds logging functionality for requests made to Google's Geming API. The logging scripts track the request and response of the API, providing useful information for debugging and analysis.

### CODE REVIEW

**Comment by jzone3** [comment]
Add it to the getting started tutorial bottom

CleanShot 2024-03-13 at 19 53 41

File: reference/track-request.mdx [hunk 1]
```diff
@@ -131,4 +129,139 @@ data_payload = {
 # Making the request
 response = requests.post("https://api.promptlayer.com/rest/track-request", json=data_payload)
 print(response.json())
-```
\ No newline at end of file
+```
+
+## Logging Google Requests
```
Comment (jzone3) [inline reference/track-request.mdx:134]: Can you add a title like this to the codeblock above for compeltions?

**Review by jzone3 — CHANGES_REQUESTED** [review]
Looks good. Let's just add another question at the bottom of the FAQ

What model providers do you support on your evaluations page?

While you can log LLM requests from any model and our Prompt Registry is agnostic, our evaluations & playground requests support OpenAI, Anthropic, Bedrock, Cohere, and Google.

File: reference/track-request.mdx (hunk 1 above)
Comment (abubakarsohail) [reply reference/track-request.mdx:134]: Sure

File: features/faq.mdx [hunk 2]
```diff
@@ -101,4 +101,8 @@ Yes, we have achieved SOC 2 Type 2 certification. Please [contact us](mailto:hel
 
 To ensure your evaluation report reflects the newest version of your prompt template, you must configure your evaluation pipeline to use the "latest" version of the prompt template in its column step. The template is fetched at runtime, and specifying a frozen version will result in the evaluation report not reflecting your newest prompt template.
 
-![Latest FAQ Updates](/images/latest-faq.png)
\ No newline at end of file
+![Latest FAQ Updates](/images/latest-faq.png)
+
+## What model providers do you support on your evaluations page?
+
+While you can log LLM requests from any model and our Prompt Registry is agnostic, our evaluations & playground requests support OpenAI, Anthropic, Bedrock, Cohere, and Google.
```
Comment (jzone3) [inline features/faq.mdx:108]: OpenAI, Anthropic, Bedrock, Cohere, and Google.

"OpenAI's GPT, Anthropic's Claude, Google's Gemini, Bedrock, Cohere."

Comment (jzone3) [reply features/faq.mdx:108]: just because I think people might command-f for gemini

**Review by jzone3 — APPROVED** [review]

File: features/faq.mdx (hunk 2 above)
Comment (abubakarsohail) [reply features/faq.mdx:108]: Good point. Updated.

**Pull request merged**